plugins/nf-wave/src/main/io/seqera/wave/plugin/config/WaveConfig.groovy

```diff
@@ -48,11 +48,13 @@ class WaveConfig {
     final private RetryOpts retryOpts
     final private HttpOpts httpClientOpts
     final private Boolean freezeMode
+    final private Boolean awaitMode
 
     WaveConfig(Map opts, Map<String,String> env=System.getenv()) {
         this.enabled = opts.enabled
         this.endpoint = (opts.endpoint?.toString() ?: env.get('WAVE_API_ENDPOINT') ?: DEF_ENDPOINT)?.stripEnd('/')
         this.freezeMode = opts.freeze as Boolean
+        this.awaitMode = opts.navigate('awaitMode', true)
```

Comment:
TODO: document `wave.awaitMode`

Reply:
on second thought, doesn't seem to have any purpose outside of container preview